.github/workflows/deploy-staging.yml

```
@@ -78,6 +78,14 @@ jobs:
       - uses: actions/checkout@v2
       - name: Setup node.js
         uses: actions/setup-node@v1
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+          role-to-assume: ${{ secrets.AWS_PROD_ROLE_TO_ASSUME }}
```

should this be staging?

---

I think it might be just: 
```suggestion
          role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
```
based off line 26